azure-samples/src/test/java/com/microsoft/azure/management/samples/ComputeSampleTests.java

```diff
@@ -86,6 +88,12 @@ public void testManageVirtualMachine() {
     }
 
     @Test
+    @Ignore("Recording fails due to a failure on Managed disks backend")
```

Comment:
As Martin has modified the scenarios/code as per new RP behavior, we should change this sample accordingly.